sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -313,131 +112,254 @@ object ScalaNativePluginInternal {
     },
     addCompilerPlugin(
       "org.scala-native" % "nscplugin" % nativeVersion cross CrossVersion.full),
-    nativeSharedLibrary := false,
-    nativeClang := None,
-    nativeClangPP := None,
+    nativeClang := {
```

Comment:
Hum this undo the work for: https://github.com/scala-native/scala-native/issues/562